orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/DeregisterInstancesFromLoadBalancerTaskSpec.groovy

```diff
@@ -43,44 +45,55 @@ class DeregisterInstancesFromLoadBalancerTaskSpec extends Specification {
 
   def "creates a deregister instances from load balancer task based on job parameters"() {
     given:
-    def operations
-    task.trafficGuard = Mock(TrafficGuard)
-    task.kato = Mock(KatoService) {
-      1 * requestOperations("abc", _) >> {
-        operations = it[1]
-        rx.Observable.from(taskId)
+      def operations
+      task.trafficGuard = Mock(TrafficGuard)
```

Comment:
This test is very similar to the AbstractLoadBalancerRegistrationTaskSpec aside from the task being used and the traffic guard. I'm wondering if some of the data and asserts should be reused.